docs/sources/_index.md

```diff
@@ -11,56 +10,55 @@ keywords:
+    - title: Pyroscope architecture
+      href: ./reference-pyroscope-architecture/
+      description: Take a deep-dive into the microservices-based architecture to learn about deployment modes, components (microservices), and more. The system has multiple horizontally scalable microservices that can run separately and in parallel.
 ---
-# Grafana Pyroscope documentation
 
-![Grafana Pyroscope](logo.png)
+{{< docs/hero-simple key="hero" >}}
 
-Grafana Pyroscope is an open source software project for aggregating continuous profiling data.
-Continuous profiling is an observability signal that allows you to understand your workload's resources (CPU, memory, etc.) usage down to the line number.
-
-Grafana Pyroscope is fully integrated with Grafana allowing you to **correlate** with other observability signals, like metrics, logs, and traces.
-
-## Continuous profiling
-
-Continuous profiling is more than just a performance analysis tool.
-It's a crucial component in modern software development and operations.
-It goes past traditional profiling techniques by providing ongoing, in-depth insights into application performance.
-
-Continuous profiling goes past the ephemeral, localized nature of traditional profiling (which historically has been more similar to `console.log` or `print statement` debugging) to a structured, centralized approach allows for effective use in production environments.
-Put more simply, Pyroscope takes you from a bunch of flamegraph files on your desktop to a database where you can query and analyze production data in a structured way.
+---
 
-{{% admonition type="tip" %}}
-To learn more about continuous profiling, read [the Introduction section]({{< relref "./introduction" >}}).
-{{% /admonition %}}
+## Overview
 
-Pyroscope, in particular, offers the flexibility to either visualize more traditional adhoc data or evolve your applications observability tooling to include more modern continuous profiling capabilities.
+Grafana Pyroscope is a multi-tenant, continuous profiling aggregation system, aligning its architectural design with Grafana Mimir, Grafana Loki, and Grafana Tempo.
+It facilitates the ingestion, storage, and querying of profiles and integrates with Grafana.
+This integration enables a cohesive correlation of profiling data with existing metrics, logs, and traces.
 
+Continuous profiling provides ongoing, in-depth insights into application performance.
+With Pyroscope, you go from a bunch of flame graph files on your desktop to a database you can query and analyze production data in a structured way.
 
-<video style="border-radius: 1%" autoplay loop>
+<video style="border-radius: 1%; width: 75%; display: block; margin-left: auto; margin-right: auto;" autoplay loop>
   <source src="ui.webm" type="video/webm">
```

Comment:
It probably makes sense to update that video. It shows a quite outdated version of explore. We probably want to show either the new explore (or the app shipped with Pyroscope).

Reply:
I replace the video with this screenshot: 
![image](https://github.com/grafana/pyroscope/assets/104772500/6a31df78-213c-43a2-9b72-05b6d3f9b3f1)